tools/slang-unit-test/unit-test-vtable-stability.cpp

```
@@ -0,0 +1,1655 @@
+    SLANG_NO_THROW SlangResult SLANG_MCALL queryInterface(SlangUUID const&, void**) SLANG_OVERRIDE
+    {
+        lastSlot = 0;
+        return SLANG_OK;
+    }
+    SLANG_NO_THROW uint32_t SLANG_MCALL addRef() SLANG_OVERRIDE
+    {
+        lastSlot = 1;
+        return 1;
+    }
+    SLANG_NO_THROW uint32_t SLANG_MCALL release() SLANG_OVERRIDE
+    {
```

Cppcheck (2.20.0)

[performance] Variable 'm_buffer' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)